ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -186,6 +187,16 @@ def __init__(
         self._timeout_wait: int = timeout_wait
         self._communicator = self._get_communicator(worker_id, base_port, timeout_wait)
         self._worker_id = worker_id
+        if side_channels is None:
+            side_channels = []
+        default_training_side_channel: Optional[
+            DefaultTrainingAnalyticsSideChannel
+        ] = None
+        if "TrainingAnalyticsSideChannel" not in [
```

Comment:
Would it be better to add a channel name property to the interfaces?

Reply:
I'm using the unique channel_id now to identify the DefaultTrainingAnalyticsSideChannel. It's safer in case we changed the class name.